### PR TITLE
feat: add jamstack arch

### PR DESCRIPTION
**Pull request by gabriel-azion**

### CODE REVIEW

File: src/content/docs/en/pages/architectures/jamstack/jamstack.mdx [hunk 1]
```diff
@@ -0,0 +1,55 @@
+---
+title: Deployment of Jamstack Websites at the Edge
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/architectures/jamstack/jamstack.mdx:2]:
```suggestion
title: Deploy Jamstack Websites at the Edge
```

File: src/content/docs/en/pages/architectures/jamstack/jamstack.mdx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+description: Improving websites capabilities with resilient, secure, and scalable solutions with Azion and Jamstack.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/architectures/jamstack/jamstack.mdx:3]:
```suggestion
description: Use Azion and Jamstack to improve websites' capabilities with resilient, secure, and scalable solutions.
```

Comment (MarianaAguilera) [reply src/content/docs/en/pages/architectures/jamstack/jamstack.mdx:3]: the other archs have a similar structure here

File: src/content/docs/en/pages/architectures/jamstack/jamstack.mdx [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+meta_tags: architecture, build, bot, development, jamstack
+namespace: docs_arch_jamstack_deploy
+permalink: /documentation/architecture/jamstack/deploy-jamstack-applications/
+---
+
+Jamstack website architectures leverage pre-rendered pages and APIs to design web applications, significantly improving e-commerce capabilities with resilient, secure, and scalable solutions. These architectures manage every part of the application, from front-end to back-end, using APIs running on JavaScript, reusable Markup templates, and prebuilt static Assets, resulting in enhanced performance, higher security, cost-effectiveness, and a superior developer experience. Meanwhile, leveraging edge computing—processing data at the network's edge closest to the user—can reduce latency and bandwidth use, further boosting performance.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/architectures/jamstack/jamstack.mdx:9]: remember to link to Lnfing Page

**Review by GabrielAzion — APPROVED** [review]

File: src/content/docs/en/pages/architectures/jamstack/jamstack.mdx [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+This solution is ideal for organizations looking to enhance website performance, improve the overall user experience, and empower developers to rapidly build and deploy high-performance, scalable applications at the edge, closer to end-users.
+
+---
+
+## Jamstack Website Architecture Diagram
+
+![Architecture diagram](/assets/docs/images/uploads/jamstack-deploy.png)
+
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/architectures/jamstack/jamstack.mdx:18]:
```suggestion
```

File: src/content/docs/en/pages/architectures/jamstack/jamstack.mdx [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+2. Developers work on application code locally on their machines. When ready, they commit changes to their local Git repository and then push these commits to a remote repository. In the case of working with Azion, this remote repository could be linked to Azion's platform to trigger further actions.
+3. The applications files are sent to Edge Storage, then a bucket is created and the files are stored in it. The developer can store data and application assets here, which can be processed quickly by the edge application.
+4. Azion Actions sets up a Continuous Deployment flow, connecting the GitHub repository with the Azion edge application created in the deployment. This process is possible through the communication between Azion Actions and Azion API.
+5. Azion Actions deploy the edge application and its cofigurations, based on a default config or the one informed by the user in a manifest file. The configs include Rules Engine, Edge Cache, Image Processor, and Edge Functions
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/architectures/jamstack/jamstack.mdx:26]:
```suggestion
5. Azion Actions deploy the edge application and its cofigurations, based on a default config or the one informed by the user in a manifest file. The configs include Rules Engine, Edge Cache, Image Processor, and Edge Functions.
```

File: src/content/docs/en/pages/architectures/jamstack/jamstack.mdx [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+6. The output from the edge node is delivered to the end-users via a domain, representing the user-facing endpoint of the web where the hosted application is accessible.
+
+---
+
+## Components 
+
+### Components
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/architectures/jamstack/jamstack.mdx:33]:
```suggestion
---
### Components
```

File: src/content/docs/en/pages/architectures/jamstack/jamstack.mdx [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+
+- [Azion Console](/en/documentation/products/guides/getting-to-know-azion-console/) is the interface where you can access all the available products and their settings..
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/architectures/jamstack/jamstack.mdx:35]:
```suggestion
- [Azion Console](/en/documentation/products/guides/getting-to-know-azion-console/) is the interface where you can access all the available products and their settings.
```

File: src/content/docs/en/pages/architectures/jamstack/jamstack.mdx [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+- [Azion CLI](/en/documentation/products/azion-cli/overview/):  an open-source command-line interface (CLI) that allows interaction with Azion Edge Platform using a terminal.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/architectures/jamstack/jamstack.mdx:36]:
```suggestion
- [Azion CLI](/en/documentation/products/azion-cli/overview/): an open-source command-line interface (CLI) that allows interaction with Azion Edge Platform using a terminal. 
```

File: src/content/docs/en/pages/architectures/jamstack/jamstack.mdx [hunk 9]
```diff
@@ -0,0 +1,55 @@
...
+---
+
+## Components 
+
+### Components
+
+- [Azion Console](/en/documentation/products/guides/getting-to-know-azion-console/) is the interface where you can access all the available products and their settings..
+- [Azion CLI](/en/documentation/products/azion-cli/overview/):  an open-source command-line interface (CLI) that allows interaction with Azion Edge Platform using a terminal. 
+- [Edge Application](/en/documentation/products/build/edge-application/): an Azion product that allows you to build applications that run in Azion's edge network.
+- [Edge Storage](/en/documentation/products/store/edge-storage/): a scalable and secure storage service designed to integrate object storage with the Azion Edge Computing Platform using the S3 standard for object operations.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/architectures/jamstack/jamstack.mdx:38]:
```suggestion
- [Edge Storage](/en/documentation/products/store/edge-storage/): a scalable and secure storage service designed to integrate object storage with the Azion Edge Platform using the S3 standard for object operations.
```

File: src/content/docs/en/pages/architectures/jamstack/jamstack.mdx [hunk 10]
```diff
@@ -0,0 +1,55 @@
...
+- Link an existing application to an Azion edge application:
+[Azion CLI link command]
+
+---
+
+## Related docs
+
+- [Azion Marketplace.](/en/documentation/products/marketplace/)
+- [Azion Edge Runtime Web APIs compatibility.](/en/documentation/devtools/runtime-apis/javascript/)
+- [Azion Edge Runtime Node.js compatibility.](/en/documentation/products/azion-edge-runtime/compatibility/node/)
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/architectures/jamstack/jamstack.mdx:55]:
```suggestion
- [Azion Marketplace](/en/documentation/products/marketplace/)
- [Azion Edge Runtime Web APIs compatibility](/en/documentation/devtools/runtime-apis/javascript/)
- [Azion Edge Runtime Node.js compatibility](/en/documentation/products/azion-edge-runtime/compatibility/node/)
```

**Review by MarianaAguilera — APPROVED** [review]

**Review by GabrielAzion — APPROVED** [review]

File: src/content/docs/pt-br/pages/arquiteturas/jamstack/jamstack.mdx [hunk 11]
```diff
@@ -0,0 +1,50 @@
+---
+title: Implantando Sites Jamstack na Borda da Rede
+description: Use Azion e Jamstack para melhorar as capacidades dos sites com soluções resilientes, seguras e escaláveis.
+meta_tags: arquitetura, construção, bot, desenvolvimento, jamstack
+namespace: docs_arch_jamstack_deploy
+permalink: /documentacao/arquitetura/jamstack/implantacao-aplicativos-jamstack/
+---
+
+As arquiteturas de sites Jamstack utilizam páginas pré-renderizadas e APIs para projetar aplicações web, aprimorando significativamente as capacidades de sites com soluções resilientes, seguras e escaláveis. Essas arquiteturas gerenciam cada parte da aplicação, da interface até o backend, usando APIs em JavaScript, templates de marcação reutilizáveis ​e ativos estáticos pré-construídos, resultando em melhor desempenho, maior segurança, custo-benefício e uma experiência superior para o desenvolvedor. Enquanto isso, a utilização do edge computing - processamento de dados no edge da rede mais próxima do usuário - pode reduzir a latência e o uso de largura de banda, aumentando ainda mais o desempenho.
```
Comment (gabriel-azion) [inline src/content/docs/pt-br/pages/arquiteturas/jamstack/jamstack.mdx:9]:
```suggestion
As arquiteturas de sites Jamstack utilizam páginas pré-renderizadas e APIs para projetar aplicações web, aprimorando significativamente as capacidades de sites com soluções resilientes, seguras e escaláveis. Essas arquiteturas gerenciam cada parte da aplicação, da interface até o backend, usando APIs em JavaScript, templates de marcação reutilizáveis ​e ativos estáticos pré-construídos, resultando em melhor desempenho, maior segurança, custo-benefício e uma experiência superior para o desenvolvedor. Enquanto isso, a utilização do edge computing - processamento de dados no edge mais próxima do usuário - pode reduzir a latência e o uso de largura de banda, aumentando ainda mais o desempenho.
```

File: src/content/docs/pt-br/pages/arquiteturas/jamstack/jamstack.mdx [hunk 12]
```diff
@@ -0,0 +1,50 @@
...
+
+Essa solução é ideal para organizações que buscam melhorar o desempenho do site, melhorar a experiência geral do usuário e capacitar os desenvolvedores a construir e implantar rapidamente aplicativos de alto desempenho e escaláveis no edge da rede, mais próximos dos usuários finais.
```
Comment (gabriel-azion) [inline src/content/docs/pt-br/pages/arquiteturas/jamstack/jamstack.mdx:11]:
```suggestion
Essa solução é ideal para organizações que buscam melhorar o desempenho do site, melhorar a experiência geral do usuário e capacitar os desenvolvedores a construir e implantar rapidamente aplicações de alto desempenho e escaláveis no edge, mais próximos dos usuários finais.
```

File: src/content/docs/pt-br/pages/arquiteturas/jamstack/jamstack.mdx [hunk 13]
```diff
@@ -0,0 +1,50 @@
...
+## Diagrama de Arquitetura Jamstack
+
+![Diagrama de Arquitetura](/assets/docs/images/uploads/jamstack-deploy.png)
+
+### Fluxo de Dados Jamstack 
+
+1. O desenvolvedor interage com a plataforma Azion usando ferramentas como o console baseado na web e a interface de linha de comando (CLI). Essas ferramentas são essenciais para configurar e gerenciar aplicativos e serviços na plataforma.
+2. Os desenvolvedores trabalham no código da aplicação localmente em suas máquinas. Quando prontos, eles confirmam as mudanças no repositório Git local e então enviam esses commits para um repositório remoto. No caso de trabalhar com a Azion, este repositório remoto pode estar vinculado à plataforma Azion para acionar ações subsequentes.
+3. Os arquivos dos aplicativos são enviados para o Edge Storage, depois é criado um bucket e os arquivos são armazenados nele. O desenvolvedor pode armazenar dados e ativos da aplicação aqui, que podem ser processados rapidamente pelo edge application.
+4. Azion Actions configura um fluxo de Implantação Contínua, conectando o repositório GitHub com  Azion criado na implantação. Esse processo é possível através da comunicação entre Azion Actions e Azion API.
```
Comment (gabriel-azion) [inline src/content/docs/pt-br/pages/arquiteturas/jamstack/jamstack.mdx:24]:
```suggestion
4. Azion Actions configura um fluxo de CI, conectando o repositório GitHub com  Azion criado na implantação. Esse processo é possível através da comunicação entre Azion Actions e Azion API.
```

File: src/content/docs/pt-br/pages/arquiteturas/jamstack/jamstack.mdx [hunk 14]
```diff
@@ -0,0 +1,50 @@
...
+- [Azion Console](/pt-br/documentacao/produtos/guias/conhecendo-o-azion-console/): a interface onde você pode acessar todos os produtos disponíveis e suas configurações.
+- [Azion CLI](/pt-br/documentacao/produtos/azion-cli/visao-geral/): uma interface de linha de comando (CLI) de código aberto que permite a interação com a plataforma edge da Azion usando um terminal.
+- [Edge Application](/pt-br/documentacao/produtos/build/edge-application/): um produto Azion que permite construir aplicações que executam no edge da Azion.
+- [Edge Storage](/pt-br/documentacao/produtos/store/edge-storage/): um serviço de armazenamento escalável e seguro projetado para integrar o armazenamento de objetos com a plataforma edge da Azion usando o padrão S3 para operações de objetos.
+
+---
+
+## Implementação 
+
+- Inicialize um edge application Jamstack com base em templates:
```
Comment (gabriel-azion) [inline src/content/docs/pt-br/pages/arquiteturas/jamstack/jamstack.mdx:39]:
```suggestion
- Inicialize uma edge application Jamstack com base em templates:
```

File: src/content/docs/pt-br/pages/arquiteturas/jamstack/jamstack.mdx [hunk 15]
```diff
@@ -0,0 +1,50 @@
...
+[Guias específicos de frameworkw](/pt-br/documentacao/produtos/build/develop-with-azion/frameworks-specific/visao-geral/)
```
Comment (gabriel-azion) [inline src/content/docs/pt-br/pages/arquiteturas/jamstack/jamstack.mdx:40]:
```suggestion
[Guias específicos de frameworkw](/pt-br/documentacao/produtos/build/develop-with-azion/frameworks-specific/visao-geral/)
```
```suggestion
[Guias específicos de framework](/pt-br/documentacao/produtos/build/develop-with-azion/frameworks-specific/visao-geral/)
```

File: src/content/docs/pt-br/pages/arquiteturas/jamstack/jamstack.mdx [hunk 16]
```diff
@@ -0,0 +1,50 @@
+---
+title: Implantando Sites Jamstack na Borda da Rede
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/arquiteturas/jamstack/jamstack.mdx:2]:
```suggestion
title: Implante Sites Jamstack no Edge da Rede
```

File: src/content/docs/pt-br/pages/arquiteturas/jamstack/jamstack.mdx [hunk 17]
```diff
@@ -0,0 +1,50 @@
...
+description: Use Azion e Jamstack para melhorar as capacidades dos sites com soluções resilientes, seguras e escaláveis.
+meta_tags: arquitetura, construção, bot, desenvolvimento, jamstack
+namespace: docs_arch_jamstack_deploy
+permalink: /documentacao/arquitetura/jamstack/implantacao-app-jamstack/
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/arquiteturas/jamstack/jamstack.mdx:6]:
```suggestion
permalink: /documentacao/arquitetura/jamstack/implantacao-aplicacoes-jamstack/
```

File: src/content/docs/en/pages/architectures/jamstack/jamstack.mdx [hunk 18]
```diff
@@ -0,0 +1,50 @@
+---
+title: Deploy Jamstack Websites at the Edge
+description: Use Azion and Jamstack to improve websites' capabilities with resilient, secure, and scalable solutions.
+meta_tags: architecture, build, bot, development, jamstack
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/architectures/jamstack/jamstack.mdx:4]:
```suggestion
meta_tags: architecture, build, development, jamstack
```

File: src/content/docs/pt-br/pages/arquiteturas/jamstack/jamstack.mdx [hunk 19]
```diff
@@ -0,0 +1,50 @@
+---
+title: Implantando Sites Jamstack na Borda da Rede
+description: Use Azion e Jamstack para melhorar as capacidades dos sites com soluções resilientes, seguras e escaláveis.
+meta_tags: arquitetura, construção, bot, desenvolvimento, jamstack
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/arquiteturas/jamstack/jamstack.mdx:4]:
```suggestion
meta_tags: arquitetura, construção, desenvolvimento, jamstack
```

File: src/content/docs/pt-br/pages/arquiteturas/jamstack/jamstack.mdx [hunk 20]
```diff
@@ -0,0 +1,50 @@
...
+namespace: docs_arch_jamstack_deploy
+permalink: /documentacao/arquitetura/jamstack/implantacao-app-jamstack/
+---
+
+As arquiteturas de sites Jamstack utilizam páginas pré-renderizadas e APIs para projetar aplicações web, aprimorando significativamente as capacidades de sites com soluções resilientes, seguras e escaláveis. Essas arquiteturas gerenciam cada parte da aplicação, da interface até o backend, usando APIs em JavaScript, templates de marcação reutilizáveis ​e ativos estáticos pré-construídos, resultando em melhor desempenho, maior segurança, custo-benefício e uma experiência superior para o desenvolvedor. Enquanto isso, a utilização do edge computing - processamento de dados no edge mais proximo do usuário - pode reduzir a latência e o uso de largura de banda, aumentando ainda mais o desempenho.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/arquiteturas/jamstack/jamstack.mdx:9]:
```suggestion
As arquiteturas de sites Jamstack utilizam páginas pré-renderizadas e APIs para projetar aplicações web, aprimorando significativamente as capacidades de sites com soluções resilientes, seguras e escaláveis. Essas arquiteturas gerenciam cada parte da aplicação, da interface até o backend, usando APIs em JavaScript, templates de Markup reutilizáveis ​e ativos estáticos pré-construídos, resultando em melhor desempenho, maior segurança, custo-benefício e uma experiência superior para o desenvolvedor. Enquanto isso, a utilização de edge computing - com um processamento de dados no edge, mais proximo do usuário - pode reduzir a latência e o uso de largura de banda, aumentando ainda mais o desempenho.
```

File: src/content/docs/pt-br/pages/arquiteturas/jamstack/jamstack.mdx [hunk 21]
```diff
@@ -0,0 +1,50 @@
...
+
+---
+
+## Diagrama de Arquitetura Jamstack
+
+![Diagrama de Arquitetura](/assets/docs/images/uploads/jamstack-deploy.png)
+
+### Fluxo de Dados Jamstack 
+
+1. O desenvolvedor interage com a plataforma Azion usando ferramentas como o console baseado na web e a interface de linha de comando (CLI). Essas ferramentas são essenciais para configurar e gerenciar applicações e serviços na plataforma.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/arquiteturas/jamstack/jamstack.mdx:21]:
```suggestion
1. O desenvolvedor interage com a plataforma da Azion usando ferramentas como o Console baseado na web e a interface de linha de comando (CLI). Essas ferramentas são essenciais para configurar e gerenciar aplicações e serviços na plataforma.
```

File: src/content/docs/en/pages/architectures/jamstack/jamstack.mdx [hunk 22]
```diff
@@ -0,0 +1,50 @@
...
+
+---
+
+## Jamstack Website Architecture Diagram
+
+![Architecture diagram](/assets/docs/images/uploads/jamstack-deploy.png)
+
+### Jamstack Website Dataflow 
+
+1. The developer interacts with Azion's platform using tools like the web-based console and the command-line interface (CLI). These tools are essential for configuring and managing applications and services on the platform.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/architectures/jamstack/jamstack.mdx:21]:
```suggestion
1. The developer interacts with Azion's platform using tools like the web-based Console and the command-line interface (CLI). These tools are essential for configuring and managing applications and services on the platform.
```

File: src/content/docs/pt-br/pages/arquiteturas/jamstack/jamstack.mdx [hunk 23]
```diff
@@ -0,0 +1,50 @@
...
+2. Os desenvolvedores trabalham no código da aplicação localmente em suas máquinas. Quando prontos, eles confirmam as mudanças no repositório Git local e então enviam esses commits para um repositório remoto. No caso de trabalhar com a Azion, este repositório remoto pode estar vinculado à plataforma Azion para acionar ações subsequentes.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/arquiteturas/jamstack/jamstack.mdx:22]:
```suggestion
2. Os desenvolvedores trabalham no código da aplicação localmente em suas máquinas. Quando prontos, eles confirmam as mudanças no repositório Git local e então enviam esses commits para um repositório remoto. No caso de trabalhar com a Azion, este repositório remoto pode estar vinculado à plataforma da Azion para acionar ações subsequentes.
```

File: src/content/docs/pt-br/pages/arquiteturas/jamstack/jamstack.mdx [hunk 24]
```diff
@@ -0,0 +1,50 @@
...
+3. Os arquivos das aplicações são enviados para o Edge Storage, depois é criado um bucket e os arquivos são armazenados nele. O desenvolvedor pode armazenar dados e ativos da aplicação aqui, que podem ser processados rapidamente pelo edge application.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/arquiteturas/jamstack/jamstack.mdx:23]:
```suggestion
3. Os arquivos das aplicações são enviados para o Edge Storage, depois é criado um bucket e os arquivos são armazenados nele. O desenvolvedor pode armazenar dados e ativos da aplicação aqui, que podem ser processados rapidamente pela edge application.
```

File: src/content/docs/pt-br/pages/arquiteturas/jamstack/jamstack.mdx [hunk 25]
```diff
@@ -0,0 +1,50 @@
...
+4. Azion Actions configura um fluxo de CI, conectando o repositório GitHub com  Azion criado na implantação. Esse processo é possível através da comunicação entre Azion Actions e Azion API.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/arquiteturas/jamstack/jamstack.mdx:24]:
```suggestion
4. Azion Actions configura um fluxo de Continuous Deployment, conectando o repositório GitHub com Azion criado na implantação. Esse processo é possível através da comunicação entre Azion Actions e Azion API.
```

File: src/content/docs/pt-br/pages/arquiteturas/jamstack/jamstack.mdx [hunk 26]
```diff
@@ -0,0 +1,50 @@
...
+5. Azion Actions implementa  e suas configurações, com base em uma configuração padrão ou na informada pelo usuário em um arquivo manifesto. As configurações incluem Rules Engine, Edge Cache, Image Processor e Edge Functions.
+6. A saída do edge node é entregue aos usuários finais através de um domínio, representando o ponto de extremidade voltado para o usuário da web onde a aplicação hospedado é acessível.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/arquiteturas/jamstack/jamstack.mdx:26]:
```suggestion
6. A saída do edge node é entregue aos usuários finais através de um domínio, representando o endpoint voltado para o usuário da web onde a aplicação hospedado é acessível.
```

File: src/content/docs/pt-br/pages/arquiteturas/jamstack/jamstack.mdx [hunk 27]
```diff
@@ -0,0 +1,50 @@
...
+
+### Componentes
+
+- [Azion Console](/pt-br/documentacao/produtos/guias/conhecendo-o-azion-console/): a interface onde você pode acessar todos os produtos disponíveis e suas configurações.
+- [Azion CLI](/pt-br/documentacao/produtos/azion-cli/visao-geral/): uma interface de linha de comando (CLI) de código aberto que permite a interação com a plataforma edge da Azion usando um terminal.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/arquiteturas/jamstack/jamstack.mdx:31]:
```suggestion
- [Azion CLI](/pt-br/documentacao/produtos/azion-cli/visao-geral/): uma interface de linha de comando (CLI) de código aberto que permite a interação com a Plataforma Edge da Azion usando um terminal.
```

File: src/content/docs/pt-br/pages/arquiteturas/jamstack/jamstack.mdx [hunk 28]
```diff
@@ -0,0 +1,50 @@
...
+- [Edge Application](/pt-br/documentacao/produtos/build/edge-application/): um produto Azion que permite construir aplicações que executam no edge da Azion.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/arquiteturas/jamstack/jamstack.mdx:32]:
```suggestion
- [Edge Application](/pt-br/documentacao/produtos/build/edge-application/): um produto Azion que permite construir aplicações que são executadas no edge da Azion.
```

File: src/content/docs/pt-br/pages/arquiteturas/jamstack/jamstack.mdx [hunk 29]
```diff
@@ -0,0 +1,50 @@
...
+- [Edge Storage](/pt-br/documentacao/produtos/store/edge-storage/): um serviço de armazenamento escalável e seguro projetado para integrar o armazenamento de objetos com a plataforma edge da Azion usando o padrão S3 para operações de objetos.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/arquiteturas/jamstack/jamstack.mdx:33]:
```suggestion
- [Edge Storage](/pt-br/documentacao/produtos/store/edge-storage/): um serviço de armazenamento escalável e seguro projetado para integrar o armazenamento de objetos com a Plataforma Edge da Azion usando o padrão S3 para operações de objetos.
```

File: src/content/docs/en/pages/architectures/jamstack/jamstack.mdx [hunk 30]
```diff
@@ -0,0 +1,50 @@
...
+- [Azion CLI](/en/documentation/products/azion-cli/overview/): an open-source command-line interface (CLI) that allows interaction with Azion Edge Platform using a terminal. 
+- [Edge Application](/en/documentation/products/build/edge-application/): an Azion product that allows you to build applications that run in Azion's edge network.
+- [Edge Storage](/en/documentation/products/store/edge-storage/): a scalable and secure storage service designed to integrate object storage with the Azion Edge Platform using the S3 standard for object operations.
+
+---
+
+## Implementation 
+
+- Initialize a Jamstack Edge Application based on templates:
+[Framework specific guides]
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/architectures/jamstack/jamstack.mdx:40]:
```suggestion
[Framework specific guides](/en/documentation/products/build/develop-with-azion/frameworks-specific/overview/)
```

File: src/content/docs/en/pages/architectures/jamstack/jamstack.mdx [hunk 31]
```diff
@@ -0,0 +1,50 @@
...
+- Link an existing application to an Azion edge application:
+[Azion CLI link command]
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/architectures/jamstack/jamstack.mdx:42]:
```suggestion
[Azion CLI link command](/en/documentation/products/cli/link-command/)
```

**Review by MarianaAguilera — APPROVED** [review]

**Pull request merged**